docs/topics/clusterdefinitions.md

```diff
@@ -486,6 +455,20 @@ Below is a list of kube-scheduler options that are _not_ currently user-configur
 
 We consider `kubeletConfig`, `controllerManagerConfig`, `apiServerConfig`, and `schedulerConfig` to be generic conveniences that add power/flexibility to cluster deployments. Their usage comes with no operational guarantees! They are manual tuning features that enable low-level configuration of a kubernetes cluster.
 
+#### Arbitary YAML for Kubernetes component manifests
+
+Arbitrary YAML specifications can be configured for kube-scheduler, kube-controller-manager, cloud-controller-manager and kube-apiserver as well. You will need to pass in a _base64_ encoded string of the kubernetes manifest YAML file to _KubernetesComponentConfig["data"]_ . For example, to pass a custom kube-scheduler config, do the following:
```

Comment:
do we only support yaml data for these 4 addons?

Reply:
These are not addons, but that mistake was probably a result of not clear documentation :)

Reply:
ah that's why 😄